### PR TITLE
2023 scratch files

### PR DESCRIPTION
**Pull request by topepo**
This renames files for different workshop structures (e.g., `advanced-day-1-scratch.R`, `advanced-day-2-scratch.R`, and `intro-scratch.R`)

### CODE REVIEW

**Comment by EmilHvitfeldt** [comment]
There is also https://github.com/tidymodels/workshops/issues/250 if you want to incorporate it into this PR

**Comment by topepo** [comment]
I will do that. Thanks

File: classwork/advanced-01-classwork.qmd [hunk 1]
```diff
@@ -25,7 +25,7 @@ hotel_rates <-
   hotel_rates %>% 
   sample_n(5000) %>% 
   arrange(arrival_date) %>% 
-  select(-arrival_date_num, -arrival_date) %>%
```
Comment (topepo) [inline classwork/advanced-01-classwork.qmd:28]: I had to change this bit for all of the setup chunks

File: classwork/advanced-02-classwork.qmd [hunk 2]
```diff
@@ -43,193 +43,221 @@ hotel_split <-
...
 
 ```{r}
-# Your code here!
-
+hotel_rec <- 
+  recipe(avg_price_per_room ~ ., data = hotel_train) %>% 
+  step_dummy(all_nominal_predictors()) %>% 
+  step_zv(all_predictors()) %>% 
+  step_normalize(all_numeric_predictors()) %>% 
+  step_spline_natural(arrival_date_num, deg_free = 10)
```
Comment (topepo) [inline classwork/advanced-02-classwork.qmd:122]: This was also added to the baseline recipe for linear regression

**Comment by topepo** [comment]
Includes updates for #250

**Review by EmilHvitfeldt — APPROVED** [review]
👍

**Pull request merged**